tests/backend_test.py

```diff
@@ -61,7 +61,7 @@ def test_quantinuum_option_params(azure_backend: AzureBackend) -> None:
     c1 = b.get_compiled_circuit(c)
     if b.is_available() and b.average_queue_time_s() < 600:
         h = b.process_circuit(c1, n_shots=1000, option_params={"error_model": False})
-        r = b.get_result(h)
+        r = b.get_result(h, timeout=1200)
```

Comment:
Do you want this timeout to be 1200 ? The other are 120

Reply:
Yes, I set it to them all to double the max reported average queue time (see three lines above).